src/pages/all-on-one-v2/all-on-one-v2.ts

```diff
@@ -113,35 +122,77 @@ export class AllOnOneV2Page {
     return this.selectedManoeuvre === key;
   }
 
-  controlledStopPress() {
-    if (this.isControlledStopDone) return;
-    const { serious, dangerous } = this.controlledStopEl;
-    if (this.controlledStopEl.faultCounter > 0) {
+  // Show me button actions
+
+  setUpShowMeButton(tellMeFault: string) {
+    const noFault = undefined;
```

Comment:
Could you put each of these mappings of fault type -> function into an object and do a single lookup instead of several ifs and returns?

Reply:
Updated to use an object with the different fault names mapping to the different functionality.